cellpack/autopack/ingredient/multi_cylinder.py

```diff
@@ -335,6 +337,31 @@ def collision_jitter(
             cylNum += 1
         return False, insidePoints, newDistPoints
 
+    def pandaMatrice(self, mat):
```

Comment:
this already exists on env (which may not be the best place for it) but we shouldn't just have the same exact function here. Probably makes more sense to make `pandaMatrix` into a static method on env since it's not using `self` at all